tests/plugins/test_resolve_composes.py

```diff
@@ -447,11 +447,18 @@ def test_inherit_parents(self, workflow, reactor_config_map, parent_compose, par
         else:
             assert set(nonearch_repourls) == set(expected_yum_repourls)
 
+        if allow_inherit and parent_compose:
+            print(caplog.text)
```

Comment:
Could we remove this print here (this is the reason CI if failing)?

Reply:
Fixed.